modules/proxy/templates/setup-proxy.tmpl

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update
+
+apt-get install -y iptables-persistent netfilter-persistent
+
+iptables -t nat -A PREROUTING -p tcp --dport ${k8s_api_port} -j DNAT --to-destination "${host}"
+iptables -t nat -A POSTROUTING -p tcp --dport ${k8s_api_port} -j MASQUERADE
+
+iptables -t nat -A PREROUTING -p tcp --dport ${cluster_assistant_port} -j DNAT --to-destination "${host}"
+iptables -t nat -A POSTROUTING -p tcp --dport ${cluster_assistant_port} -j MASQUERADE
+
+netfilter-persistent save
+
+echo 1 > /proc/sys/net/ipv4/ip_forward
+
+echo "net.ipv4.ip_forward=1" >> /etc/sysctl.conf
```

Comment:
just to validate, we're intending to leave this pinned to ubuntu and not let a user update it?

https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/50/files#diff-7b02f38b90d14c1ed3783942df2238e8R69

Reply:
I think it makes sense to consider that change but we should address it in a separate issue since it will require some extra attention beyond this simple restructuring.